sparsity/test/test_sparse_frame.py

```diff
@@ -877,6 +883,69 @@ def test_drop_multiple_labels():
     np.testing.assert_array_equal(sf.data.todense(), correct)
 
 
+def test_label_based_indexing_col(sample_frame_labels):
+    key = ['A', 'B']
+    results = [
+        sample_frame_labels[key],
+        sample_frame_labels.loc[:, key],
+        sample_frame_labels.reindex(columns=key)
+    ]
+    for res in results:
+        np.testing.assert_array_equal(
+            res.data.todense(), np.identity(5)[:, :2])
+        assert (res.index == pd.Index(list('VWXYZ'))).all()
+        assert (res.columns == pd.Index(list('AB'))).all()
+
+
+def test_label_based_indexing_idx(sample_frame_labels):
+    key = ['X', 'Y', 'Z']
+    results = [
+        sample_frame_labels.loc[key],
```

Comment:
Just to be sure, could you please check also `.loc[key, :]`?

Reply:
done